app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+  def cart_count_over_one
+    if total_cart_items > 0
+      return "<span class='tag is-dark'>#{total_cart_items}</span>".html_safe
```

Comment:
no need return keyword

Reply:
also change to this `"<span class='tag is-dark'>#{total_cart_items}</span>".html_safe if total_cart_items > 0`